nilearn/plotting/html_report.py

```diff
@@ -0,0 +1,44 @@
+import os
+import tempita
+from . import js_plotting_utils as plot_utils
+
+
+class ReportMixin():
```

Comment:
classes need to be derived from `object`. I'm not entirely sure we need to define a class though

Reply:
@jeromedockes That's no longer necessary. In python3, classes don't have to be derived from anything.
@emdupre The parantheses are unnecessary as well.

Reply:
good to know, thanks!